File: tkpddiscovery/src/main/java/com/tokopedia/discovery/similarsearch/model/Shop.java
package com.tokopedia.discovery.similarsearch.model;

import com.google.gson.annotations.SerializedName;

public class Shop{

	@SerializedName("city")
	private String city;

	@SerializedName("name")
	private String name;

	@SerializedName("id")
	private int id;

	@SerializedName("is_gold_shop")
	private boolean isGoldShop;

	public void setCity(String city){
		this.city = city;
	}

	public String getCity(){
		return city;
	}

	public void setName(String name){
		this.name = name;
	}

	public String getName(){
		return name;
	}

	public void setId(int id){
		this.id = id;
	}

	public int getId(){
		return id;
	}

	public boolean isGoldShop() {
		return isGoldShop;
	}

	public void setGoldShop(boolean goldShop) {
		isGoldShop = goldShop;
	}

	@Override
 	public String toString(){
		return 
			"Shop{" + 
			"city = '" + city + '\'' + 
			",name = '" + name + '\'' + 
			",id = '" + id + '\'' + 
			"}";
		}
}

File: features/discovery/similarsearch/src/main/java/com/tokopedia/similarsearch/Shop.kt
package com.tokopedia.similarsearch

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

internal data class Shop(
        @SerializedName("id")
        @Expose
        val id: Int = 0,

        @SerializedName("is_gold_shop")
        @Expose
        val isGoldShop: Boolean = false,

        @SerializedName("location")
        @Expose
        val location: String = "",

        @SerializedName("city")
        @Expose
        val city: String = "",

        @SerializedName("reputation")
        @Expose
        val reputation: String = "",

        @SerializedName("clover")
        @Expose
        val clover: String = "",

        @SerializedName("is_official")
        @Expose
        val isOfficial: Boolean = false
)